db/migrations/V2.15.0__Add_rakennuksen_vanhimmat.sql

```diff
@@ -0,0 +1,43 @@
+-- object: jkr.rakennuksen_vanhimmat | type: TABLE --
+-- DROP TABLE IF EXISTS jkr.rakennuksen_vanhimmat CASCADE;
+CREATE TABLE jkr.rakennuksen_vanhimmat (
+	rakennus_id integer NOT NULL,
+	osapuoli_id integer NOT NULL,
+    huoneistokirjain text,
+    huoneistonumero integer,
+    jakokirjain text,
+	CONSTRAINT rakennuksen_vanhimmat_pk PRIMARY KEY (rakennus_id,osapuoli_id)
```

Comment:
Our primary key can't be just rakennus_id, osapuoli_id becaus that is not unique anymore when we added those additional fields. Probably just a normal id here is the cleanest solution.

Reply:
Good point, missed that completely!

Reply:
Going to go for just (osapuoli_id) pk here, since people can only have one permanent address. Plus some extra uniqueness constraints for additional field combinations.